ouroboros-consensus-protocol/src/ouroboros-consensus-protocol/Ouroboros/Consensus/Protocol/Praos.hs

```diff
@@ -333,6 +335,7 @@ instance Serialise PraosState where
         <*> fromCBOR
         <*> fromCBOR
         <*> fromCBOR
+        <*> pure SNothing
         <*> fromCBOR
         <*> fromCBOR
```

Comment:
I assumed that storing the previous epoch nonce is something we only want to do in memory and not on disk. Is this correct?